gigs/gigs/ViewController/GigsTableViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  GigsTableViewController.swift
+//  gigs
+//
+//  Created by Taylor Lyles on 5/16/19.
+//  Copyright © 2019 Taylor Lyles. All rights reserved.
+//
+
+import UIKit
+
+class GigsTableViewController: UITableViewController {
+	
+	let gigController = GigController()
+
+    override func viewDidLoad() {
```

Comment:
I would do this on ViewDidAppear, you also aside from reloading the table view you need to fetch all gigs from your model controller.

```
override func viewDidAppear(_ animated: Bool) {
		super.viewDidAppear(animated)
		
		//tableView.reloadData()
        
        if gigController.bearer == nil {
            performSegue(withIdentifier: "LoginModal", sender: self)
        } else {
        
        gigController.gettingGig { (error) in
            if let error = error {
                NSLog("Error getting all gigs: \(error.localizedDescription)")
            }
            
            DispatchQueue.main.async {
                self.tableView.reloadData()
            }
        }
        }
	}
```